services/showcase/frontend/src/services/showcaseService.js

```diff
@@ -24,16 +24,28 @@ export default {
   },
 
   /**
-   * @function createInvitation
-   * Create a invitation
+   * @function createInvitationStudent
+   * Create a invitation for a student
    * @param {string} sandboxId The identifier for the sandbox
    * @param {string} lobId The identifier for the line of business
+   * @param {string} studentId The identifier for the student
    * @returns {Promise} An axios response
    */
-  createInvitation(sandboxId, lobId, studentId) {
+  createInvitationStudent(sandboxId, lobId, studentId) {
     return appAxios().post(`${ApiRoutes.SANDBOXES}/${sandboxId}/lobs/${lobId}/create-invitation/student`, { student_id: studentId });
```

Comment:
Is this a generic method `createInvitation` or is this a faber specific method 'CreateInvitationtoAlice'... Just seems like it could be either and this is kind of a mix. 

Maybe just update the comments/local variables describing both roles. If someone passes in ACME's LobID to this method... there will be issues.. or at least 'undocumented behaviour'

Same for the method below. What is different between these two endpoints? Does the backend enforce the that the lobID has the intended role?

Reply:
Yeah there's the 2 invitation type endpoints on the showcase API. I don't think the backend does LOB id checking or anything...
I'll rename the variables faberLobId and acmeLobId maybe?

Reply:
Have some changes planned to clean up this file upcoming in next batch of work, so will include this change with those in a next PR.